source/features/clean-notifications.css

```diff
@@ -1,12 +1,15 @@
 /* Place conversation number before the title */
+.js-notifications-list [id^='notification'],
 .js-notifications-group [id^='notification'] {
 	flex-direction: row !important;
 }
 /* Hide duplicate repository URL */
+.js-notifications-list [id^='notification'] > .d-flex > .f6,
 .js-notifications-group [id^='notification'] > .d-flex > .f6 {
```

Comment:
`clean-notifications` is specifically for the new notifications UI; the selectors must be replaced, not added